cocos/core/gfx/device-manager.ts

```diff
@@ -155,12 +161,6 @@ export class DeviceManager {
             return;
         }
 
-        const swapchainInfo = new SwapchainInfo(this._canvas!);
-        const windowSize = screen.windowSize;
-        swapchainInfo.width = windowSize.width;
-        swapchainInfo.height = windowSize.height;
-        this._swapchain = this._gfxDevice.createSwapchain(swapchainInfo);
```

Comment:
Doesn't `HEADLESS ` mode need to create swapchain?

Reply:
Editor needs creator a swapchain，but native platforms like Windows doesn't.
I'm not sure where these codes are better.